packages/context/src/context.ts

```diff
@@ -21,12 +28,33 @@ export class Context {
    * Name of the context
    */
   readonly name: string;
+
+  /**
+   * Key to binding map as the internal registry
+   */
   protected readonly registry: Map<string, Binding> = new Map();
+
+  /**
+   * Parent context
+   */
   protected _parent?: Context;
 
   /**
-   * Create a new context
+   * A list of registered context listeners
+   */
+  public readonly listeners: Set<ContextEventListener> = new Set();
```

Comment:
IMO, `listeners` should be a private implementation detail. By making this property public, we make it more difficult to change the underlying implementation in the future.

Reply:
I made it public for test assertions. Now I added a `isSubscribed` method instead.